### ISSUE TITLE
how to arrange the directory of datasets 

### ISSUE THREAD

**Issue by logeek404** [piotrkawa/audio-deepfake-adversarial-attacks#4]
I want to train the model using asvspoof dataset . I already downloaded all of them including DF subsets. However whenever I tried to run .There is always error around the path(dir) configuration. Just want to know how to arrange them to make it work.

**Comment by piotrkawa** [comment]
Hi,
could you provide a detailed stacktrace? One of the possible errors you could get would be because at the time of the paper being written there were no exact labels for ASVSpoof2021 DF, so we based on this [keys&metadata file](https://www.asvspoof.org/resources/DF-keys-stage-1.tar.gz).

Could you please try running code with these labels?

**Comment by logeek404** [comment]
> Hi, could you provide a detailed stacktrace? One of the possible errors you could get would be because at the time of the paper being written there were no exact labels for ASVSpoof2021 DF, so we based on this [keys&metadata file](https://www.asvspoof.org/resources/DF-keys-stage-1.tar.gz).
> 
> Could you please try running code with these labels?
Yeah , I tried it . But there's another error after loading the dataset .
![image](https://github.com/piotrkawa/audio-deepfake-adversarial-attacks/assets/106349957/2c2d88d2-1d0a-4a10-a2d2-e60e5b081f90)

![image](https://github.com/piotrkawa/audio-deepfake-adversarial-attacks/assets/106349957/6a834558-2677-4826-9561-dc80023c44e9)
I have tried it on python 3.10 and python 3.9 with pandas == 1.2.4
Does that mean that I have to download all three datasets ( FAKEVAV, WaveFake, ASVSPOOF) to run it  cause I only downloaded the asvspoof df subset

**Comment by piotrkawa** [comment]
Could you please provide information on where exactly `KeyError: 'attack type'` happens? It is used in couple of places and I cannot find its location in this stacktrace.

To only use ASVspoof you have to set proper paths (`wavefake_path` and `fakeavceleb_path`) as None - please refer to [this function](https://github.com/piotrkawa/audio-deepfake-adversarial-attacks/blob/main/src/datasets/detection_dataset.py#L70).
